app/install/source.md

```diff
@@ -36,6 +34,8 @@ redirect_from: /install/compile/
     $ ./configure --with-pcre-jit --with-ipv6 --with-http_realip_module --with-http_ssl_module --with-http_stub_status_module
     ```
 
+    Install [Luarocks v{{luarocks_version}}](https://github.com/keplerproject/luarocks/wiki/Download) (note: configure to run with the Lua/LuaJit interpreter included with OpenResty/Nginx)
```

Comment:
OpenResty includes LuaJit 2.0.4. I think we need stronger wording to encourage people to use the OpenResty included one, to prevent potential bugs or test failures from running on different executables.